wire-ios/Wire-iOS/Sources/UserInterface/Settings/DeviceManagement/ClientListViewController.swift

```diff
@@ -450,6 +491,64 @@ final class ClientListViewController: UIViewController,
         navigationItem.setupNavigationBarTitle(title: L10n.Localizable.Registration.Devices.title.capitalized)
     }
 
+    @MainActor
+    private func updateCertificates(for userClients: [UserClient]) async -> [UserClient] {
```

Comment:
@theswiftway  This method is difficult to follow because of the many “if else.” I would simplify and move the code into helper methods. This will make our life easier when we need to debug or test the logic. 
@netbe @johnxnguyen what do you think?

Reply:
https://wearezeta.atlassian.net/browse/WPB-6798